models/in_tcpd_elections/code/upload.py

```
@@ -0,0 +1,104 @@
+    path = OUTPUT_ROOT / table_id
+    if not path.exists():
+        raise FileNotFoundError(f"Missing output path: {path}")
+
+    tb = bd.Table(dataset_id=dataset_id, table_id=table_id)
+
+    # Delete stale GCS staging prefix before upload (avoids partition conflicts)
+    st = bd.Storage(dataset_id=dataset_id, table_id=table_id)
+    try:
+        st.delete_table(mode="staging", not_found_ok=True)
+        print("  staging prefix: cleared", flush=True)
+    except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)